src/shared/mount-util.c

```diff
@@ -78,10 +78,8 @@ int umount_recursive_full(const char *prefix, int flags, char **keep) {
                         if (!path)
                                 continue;
 
-                        if (prefix && !path_startswith(path, prefix)) {
-                                log_trace("Not unmounting %s, outside of prefix: %s", path, prefix);
```

Comment:
> There's probably a deeper issue to fix here, but for now, let's drop the logging message to make sure test-execute doesn't become impossibly slow due to all the logging.

Then, comment this out with a FIXME instead of removing it wholly?